shell/platform/common/cpp/text_input_model.cc

```diff
@@ -26,10 +26,16 @@ static constexpr char kTextInputAction[] = "inputAction";
 static constexpr char kTextInputType[] = "inputType";
 static constexpr char kTextInputTypeName[] = "name";
 
+#if defined(_MSC_VER)
+// TODO(naifu): This temporary code is to solve link error.(VS2015/2017)
+// https://social.msdn.microsoft.com/Forums/vstudio/en-US/8f40dcd8-c67f-4eba-9134-a19b9178e481/vs-2015-rc-linker-stdcodecvt-error
+std::locale::id std::codecvt<char16_t, char, _Mbstatet>::id;
+#endif  // defined(_MSC_VER)
+
 namespace flutter {
 
 TextInputModel::TextInputModel(int client_id, const rapidjson::Value& config)
-    : text_(""),
+    : text_(std::u16string()),
```

Comment:
You can just remove this line; the default constructor will give you an empty string.